### PR TITLE
Added a new connector file for AMcards.com.

### PR DESCRIPTION
**Pull request by teewuane**
Please make this connector available to all users.

### CODE REVIEW

File: oauth2/amcards_connector.rb [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+          },
+        ]
+      },
+    },
+  },
+
+  actions: {
+    create_card: {
+      input_fields: ->(object_definitions) {
+        object_definitions['card'].required('template_id', 'initiator', 'first_name', 'last_name', 'address_line_1', 'city', 'state', 'postal_code', 'return_first_name', 'return_last_name', 'return_address_line_1', 'return_city', 'return_state', 'return_postal_code')
```
Comment (eeshan92) [inline oauth2/amcards_connector.rb]: fields that have `optional: false` in the object_definition is by default required, so there is no need to include them in `.required()`

File: oauth2/amcards_connector.rb [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+            type: :integer,
+            hint: "You must own the template or it must be public in order to use it.",
+            control_type: 'select',
+            pick_list: 'templates',
+            optional: false,
+          },
+          {
+            name: 'message',
+            label: 'Message for inside of card',
+            # control_type: 'textarea',
```
Comment (eeshan92) [inline oauth2/amcards_connector.rb]: you can use 'text-area' here

**Comment by eeshan92** [comment]
@teewuane Hi, this is excellent work! Added some comments to be address. Looking forward to it adding this as an example.

**Comment by teewuane** [comment]
@eeshan92 Thanks! I've pushed those changes up. Should I now create a new pull request?

**Comment by eeshan92** [comment]
@teewuane No need, this will do.

**Pull request merged**